### DISCUSSION TITLE
CrazyS did not take off after launching the hovering example.

### DISCUSSION THREAD

**Discussion by hoangah2000vn** [gsilano/CrazyS#98] [General]
I launched the following command
`$ roslaunch rotors_gazebo crazyflie2_hovering_example.
launch enable_state_estimator:=true`
The gazebo interace pops up but the crazyflie did not take off after 5 seconds. It seems as though the trajectory was not published/received. What could be happening here? How do I fix it?

**Comment by gsilano** [comment]
Hi @hoangah2000vn! This issue has already been addressed in #92, #90, #85, #79, #77 and #50. If you still have trouble in make It work, don't hesitate to give me back.
